GooglePlayInstant/Samples/SphereBlast/scripts/AnimateLoadingText.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using UnityEngine;
+
+public class AnimateLoadingText : MonoBehaviour {
```

Comment:
Going to try to merge Quick Deploy today or tomorrow. Will be superseded.

Reply:
I'm inclined to merge this PR tomorrow, and then do another PR with the changes after quick deploy is integrated. WDYT?